apps/testingaccessibility/src/components/pricing.tsx

```diff
@@ -72,20 +77,28 @@ export const Pricing: React.FC<PricingProps> = ({
         }),
   )
 
-  const availableCoupon = formattedPrice?.availableCoupons?.[0]
+  const availableCoupon = first(formattedPrice?.availableCoupons)
+
   const appliedMerchantCoupon = formattedPrice?.appliedMerchantCoupon
+
   const fullPrice =
     (formattedPrice?.unitPrice || 0) * (formattedPrice?.quantity || 0)
+
   const percentOff =
     appliedMerchantCoupon &&
     Math.floor(appliedMerchantCoupon.percentageDiscount * 100)
+
   const percentOffLabel =
     appliedMerchantCoupon && `${percentOff}% off of $${fullPrice}`
-  const pppCoupon = find(
-    formattedPrice?.availableCoupons,
-    (coupon) => coupon.type === 'ppp',
-  )
+
+  const pppCoupon = getFirstPPPCoupon(formattedPrice?.availableCoupons)
+
+  // if there is no available coupon, hide the box (it's not a toggle)
+  // only show the box if ppp coupon is available
+  // do not show the box if purchased
+  // do not show the box if it's a downgrade
   const showPPPBox =
+    availableCoupon &&
```

Comment:
explained in the comment, but this wasn't meant to act as a toggle. Generally we only show the PPP discount if the discount is more than the active discount, so they shouldn't need to decide one way or the other, but page refresh is the "toggle". Might make sense to use a different UI element?

either way, it should go way and not linger as a choice.